docs/conf.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+#
+# Hypothesis Client documentation build configuration file, created by
+# sphinx-quickstart on Thu Mar  9 18:01:59 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import os
+# import sys
+
+from recommonmark.parser import CommonMarkParser
```

Comment:
It looks like you've translated all the docs from Markdown to rST. In that case we could leave out the markdown support for the moment.